templates/classpath-jars-pv.yaml

```diff
@@ -35,7 +35,7 @@ metadata:
     type: amazonEFS
     {{- end }}
   annotations:
-    "helm.sh/hook": "pre-install"
+    "helm.sh/hook": "pre-install, pre-upgrade"
```

Comment:
What was the conclusion on this? If we have it pre-upgrade do we also need a hook-delete-policy?